.github/workflows/stable.yml

```diff
@@ -40,10 +45,12 @@ jobs:
         MESSAGE: 'Release: ({sha}) {msg}'
 
     - name: Build docs
-      run: npm run build:docs
+      run: |
+        rm -rf ./release
+        npm run build:docs
 
     - name: Deploy docs update
       uses: peaceiris/actions-gh-pages@4f9cc6602d3f66b9c108549d475ec49e8ef4d45e # v4.0.0
       with:
         github_token: ${{ secrets.GITHUB_TOKEN }}
-        publish_dir: './docs'
+        publish_dir: './docs/build'
```

Comment:
So right now our documentation site is at https://10up.github.io/classifai/. Will that URL be different after this or does it stay the same?